Add OCO Vanilla Hair - Version FR

YokiHotsumi opened the pull request:
* Modify plugin name:
- OCO Cleaned - Vanilla Hair.esp
to
- (OCO Cleaned - Vanilla Hair|OCO Vanilla Hair)\.esp

* Add new location
- OCO Vanilla Hair - Version FR:
https://www.nexusmods.com/oblivion/mods/51998/
- Add mod name.

* Add cleaning data
- OCO Vanilla Hair - Version FR v1.2

- OCO Vanilla Hair.esp:
- OCO Vanilla Hair - Cleaned: B1E49BA0.
- OCO Vanilla Hair - PNOO: 3291418D.

YokiHotsumi committed:
Update OCO Cleaned - Vanilla Hair

* Modify plugin name:
- OCO Cleaned - Vanilla Hair.esp
to
- (OCO Cleaned - Vanilla Hair|OCO Vanilla Hair)\.esp

* Add new location
- OCO Vanilla Hair - Version FR:
https://www.nexusmods.com/oblivion/mods/51998/
- Add mod name.

* Add cleaning data
- OCO Vanilla Hair - Version FR v1.2

- OCO Vanilla Hair.esp:
- OCO Vanilla Hair - Cleaned: B1E49BA0.
- OCO Vanilla Hair - PNOO: 3291418D.

sibir-ine commented on masterlist.yaml:
```diff
@@ -4403,7 +4403,7 @@ plugins:
         util: 'TES4Edit v4.0.4b'
   - name: 'Improved NPC Faces for OCOv2.esp'
-    after: [ 'OCO Cleaned - Vanilla Hair.esp' ]
+    after: [ '(OCO Cleaned - Vanilla Hair|OCO Vanilla Hair)\.esp' ]
```
Regexes aren't supported for the `after` key, so you have to explicitly enumerate each plugin (for future reference, LOOT's metadata syntax is [fully documented](https://loot-api.readthedocs.io/en/stable/metadata/data_structures/plugin.html)):

```suggestion
    after:
      - 'OCO Cleaned - Vanilla Hair.esp'
      - 'OCO Vanilla Hair.esp'
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -4662,14 +4662,21 @@ plugins:
       - crc: 0x07E94F26
         util: 'TES4Edit v4.0.4b'
-  - name: 'OCO Cleaned - Vanilla Hair.esp'
+  - name: '(OCO Cleaned - Vanilla Hair|OCO Vanilla Hair)\.esp'
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/51324/'
         name: 'Oblivion Character Overhaul Cleaned - OCO Cleaned'
+      - link: 'https://www.nexusmods.com/oblivion/mods/51998/'
+        name: 'OCO Vanilla Hair - Version FR'
     msg:
       - <<: *wildEdits
         subs: [ 'Remove **0001D9D8**, **000234B7**, & **00034E7E** from **OCO Cleaned - Vanilla Hair.esp**.' ]
         condition: 'checksum("OCO Cleaned - Vanilla Hair.esp", CF743DF7)'
+    clean:
+      - crc: 0xB1E49BA0
+        util: 'TES4Edit v4.0.4b'
+      - crc: 0x3291418D
+        util: 'TES4Edit v4.0.4b'
```
`OCO Vanilla Hair.esp` is hosted on a separate page that also doesn't host `OCO Cleaned - Vanilla Hair.esp`, so it'd be more appropriate to separate these plugins. You can place the new plugin just after `OCO uses merged teeth.esp` in the masterlist (i.e., alphanumerically sorted), making sure there's a single extra newline between the two and another after it to keep it properly separated.

The new plugin would look like this:

```yaml
  - name: 'OCO Vanilla Hair.esp'
    url:
      - link: 'https://www.nexusmods.com/oblivion/mods/51998/'
        name: 'OCO Vanilla Hair - Version FR'
    clean:
      - crc: 0xB1E49BA0
        util: 'TES4Edit v4.0.4b'
      - crc: 0x3291418D
        util: 'TES4Edit v4.0.4b'
```

YokiHotsumi replied:
Hello Sibir,

I understand. I'm sorry I'm still learning. I will make the necessary changes. Thanks for your help.

sibir-ine replied:
No worries, that's what the review process is for. You're welcome, & thank you for contributing!

YokiHotsumi replied:
I made the changes, hoping I didn't make a mistake.

YokiHotsumi committed:
Add OCO Vanilla Hair

* Add new location
- OCO Vanilla Hair - Version FR:
https://www.nexusmods.com/oblivion/mods/51998/
- Add mod name.

* Add cleaning data
- OCO Vanilla Hair - Version FR v1.2

- OCO Vanilla Hair.esp:
- OCO Vanilla Hair - Cleaned: B1E49BA0.
- OCO Vanilla Hair - PNOO: 3291418D.

* Modify load order
- Improved NPC Faces for OCOv2.esp
Must be load after
- OCO Cleaned - Vanilla Hair.esp
- OCO Vanilla Hair.esp

sibir-ine reviewed (approved):
Perfect, thanks!
